utils/payload_utils.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+class PayloadUtils:
+
+    def extract_testing_status(description):
```

Comment:
I assume this would be our way of parsing out key:value pairs from the Milestone textarea in a modular way, correct?

Reply:
yes